cscs-checks/apps/amber/amber_check.py

```diff
@@ -43,10 +43,10 @@ def __init__(self, **kwargs):
         self.executable = 'pmemd.cuda.MPI'
         self.reference = {
             'dom:gpu': {
-                'perf': (22.0, -0.05, None)
+                'perf': (22.16, -0.05, None)
```

Comment:
I think that `22.0` is the same as `22.16`, so I would leave it as `22.0`. What do you think?